sams_mansion/map.conf

```
@@ -0,0 +1,27 @@
+team.1.pos = (-50,0,9)
+team.1 = red
+team.2.color = blue
+team.1.color = red
+team.2 = blue
+team.2.pos = (46,-7,-5)
+h = 26
+r = 62
+rotation = z
+author = Xenon
+name = Sam's Mansion
+initial_stuff = default:pick_steel,default:desert_sandstone 50,default:torch 10,shooter:pistol
```

```suggestion
initial_stuff = default:pick_steel,default:desert_sandstone 50,default:torch 10,shooter_guns:pistol
```

---

Its should be 
```suggestion
initial_stuff = default:pick_steel,default:desert_sandstone 50,default:torch 10,shooter_guns:pistol_loaded
```
```properties
initial_stuff = default:pick_steel,default:desert_sandstone 50,default:torch 10,shooter_guns:pistol_loaded
```
Else it will give an unloaded pisol.

---

`shooter:pistol` works fine. Don't change it.